app/models/operatingsystems/vrp.rb

```diff
@@ -0,0 +1,61 @@
+class VRP < Operatingsystem
+  PXEFILES = {:core => "firmware.cc", :web => "web.7z"}
+
+  # Simple output of the media url
+  def mediumpath(medium_provider)
+    medium_provider.medium_uri.to_s
+  end
+
+  def pxe_prefix(medium_provider)
+    "ztp.cfg/images/#{medium_provider.unique_id}/"
+  end
+
+  # The PXE type to use when generating actions and evaluating attributes. jumpstart, kickstart and preseed are currently supported.
+  def pxe_type
+    "ZTP"
+  end
+
+  def available_loaders
+    ["None"]
```

Comment:
is that purely a UI thing? it might be better to return an empty array instead?

Reply:
Hey @ohadlevy , I actually used the code from JunOS as inspiration in which such code is also used. If you like, I can make it an empty array but I'll have to validate any implications this might have (at least, from my end, I don't use this at all)

Reply:
Hi @ohadlevy , just gave it a spin with an empty list of loaders, however, it doesn't work. Looking more closely, it seems the system checks to see if a pxe_loader has been selected:
https://github.com/theforeman/foreman/blob/develop/app/models/concerns/orchestration/tftp.rb#L18

I also noticed that NXOS & Solaris do it like that, probably for the same reason.

IMHO if we want to tackle this, this should be a different issue which deals with the implications of altering that check, no? Unless you see a harmless way forward?

Reply:
This was designed that way - PXELoader is a string and "None" is one of the allowed options, we check for this value specifically. Looks good to me.